pkg/utils/quantity_test.go

```diff
@@ -0,0 +1,39 @@
+package utils
```

Comment:
Reuse of https://github.com/kubernetes/apimachinery/blob/master/pkg/api/resource/quantity.go#L125 looks like it would negate this test

Reply:
I didn't follow  what you mean ? I am testing **ParseQuantity** method of quantity.go from utils package. Also I didn't write quantity.go and quantity_test.go code. I took  code from latest master branch

Reply:
Reusing the existing library means this test is not required. I'm not certain why this is in the master branch but IMO it should be removed from there as well

Reply:
This file is removed now.